Program.cs

```diff
@@ -721,103 +722,117 @@ public static void Props()
+                Console.WriteLine("Would you like to attack or defend?");
+                temp = Console.ReadLine().ToLower();
+                Console.Clear();
+                switch (temp)
                 {
                     case "attack":
-                        if ((computer == 0) || (computer == 1) || (computer == 2) || (computer == 3))
+                        if (computerAction == 0) //Both attack
                         {
-                            playerHP = playerHP - computeratt;
-                            computerHP = computerHP - playeratt;
-                            Console.WriteLine("you choose attack");
-                            Console.WriteLine($"Enemy attack is {computeratt:D2}");
-                            Console.WriteLine($"your hp is {playerHP:D2}");
-                            Console.WriteLine($"Enemy hp is {computerHP:D2}");
+                            toHit = rand.Next(0, 4);
+                            if (toHit >= 1)
+                            {
+                                Console.WriteLine("You strike your enemy and hit a solid blow");
+                                compHP -= playerDMG;
+                            }
+                            else
+                            {
+                                Console.WriteLine("You swing and miss");
+                            }
+                            toHit = rand.Next(0, 4);
+                            if (toHit >= 1)
+                            {
+                                Console.WriteLine("Your enemy strikes you and hits a solid blow");
+                                playerHP -= compDMG;
+                            }
+                            else
+                            {
+                                Console.WriteLine("Your enemy swings and misses");
+                            }
                         }
-                        else
+                        else //Human attack, computer defend
                         {
-                            playerHP = playerHP - 0;
-                            computerHP = computerHP - 0;
-                            Console.WriteLine("The enemy resisted the attack");
-                            Console.WriteLine($"your hp is {playerHP:D2}");
-                            Console.WriteLine($"Enemy hp is {computerHP:D2}");
+                            toHit = rand.Next(0, 4);
+                            if (toHit >= 2)
+                            {
+                                Console.WriteLine("You strike your enemy and hit a glancing blow");
+                                compHP -= playerDMG / 2;
+                            }
+                            else
+                            {
+                                Console.WriteLine("You swing and miss");
+                            }
                         }
                         break;
                     case "defend":
```

Comment:
you could allow for short hand code as well for both attack and defend

Reply:
"a" and "atk" now work for attack and "d" and "def" now work for defend.